client/web/compose/src/components/PageBlocks/CommentBase.vue

```diff
@@ -5,84 +5,94 @@
     @refreshBlock="refresh"
   >
     <div
-      v-if="processing"
-      class="d-flex align-items-center justify-content-center h-100"
+      v-if="!contentField"
```

Comment:
You should turn this into a isConfigured computed. Since title field is also required no?
So it deserves a * aswell